### PR TITLE
Remove unnecessary test on $service_ensure

### PR DESCRIPTION
**Pull request by chrekh**
$service_ensure is required by init.pp to have type Stdlib::Ensure::Service, so there is
no need for this test.

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -7,10 +7,6 @@
   $service_manage = $chrony::service_manage,
   $service_name   = $chrony::service_name,
 ) inherits chrony {
```
Comment (igalic) [inline manifests/service.pp:9]: why does this class inherit from `chrony`
that's really bad form for anything other than the params class

Comment (chrekh) [reply manifests/service.pp:9]: I have no idea. It has been that way since commit 74c0dc8 when the the file was added.

Comment (igalic) [reply manifests/service.pp:9]: okay
let's remove those in a future PR

i'm pretty, and sure, there's a standard puppet lint that should be complaining about this

have we gotten module/pdk sync yet for this module?

**Comment by chrekh** [comment]
Oops. I will have to change the tests also. Don't merge until I have fixed that.

**Pull request merged**